drop-boxes/register-qpcr-dropbox/register-qpcr-dropbox.py

```diff
@@ -26,10 +26,10 @@
 SERVICE_CREDENTIALS.user = tracking_helper.get_service_user()
 SERVICE_CREDENTIALS.password = tracking_helper.get_service_password()
 SERVICE_REGISTRY_URL = URL(tracking_helper.get_service_reg_url())
-QBIC_LOCATION = tracking_helper.get_qbic_location_json()
+DATA_AVAILABLE_JSON = tracking_helper.get_data_available_status_json()
 
-### We need this object to update the sample location later
-SAMPLE_TRACKER = SampleTracker.createQBiCSampleTracker(SERVICE_REGISTRY_URL, SERVICE_CREDENTIALS, QBIC_LOCATION)
+### We need this object to update the sample status later
+SAMPLE_TRACKER = SampleTracker.createLocationIndependentSampleTracker(SERVICE_REGISTRY_URL, SERVICE_CREDENTIALS, DATA_AVAILABLE_JSON)
```

Comment:
In general: why do we need to provide the single status here for setting up the sample tracker?

Reply:
good point, this is because it was implemented quite similar to the qbic sample tracker with its location...would need to change it in the lib, then...should I?